PendingReleaseNotes.md

```diff
@@ -10,6 +10,8 @@ v1.5...
 
 ### Ceph
 
+- CephCluster CRD now supports setting the CRUSH map root, see [ceph-cluster crd](Documentation/ceph-cluster-crd.html).
```

Comment:
If we're going to backport this to v1.4, please actually revert the pending release notes. In patch releases we create pending release notes for individual commits instead of looking at this file.